src/Chainweb/Pact/Backend/Types.hs

```diff
@@ -190,7 +182,7 @@ type SQLitePendingTableCreations = HashSet ByteString
 type SQLitePendingSuccessfulTxs = HashSet ByteString
 
 -- | Pending writes to the pact db during a block, to be recorded in 'BlockState'.
```

Comment:
i think you should document that the outer bytestring is the table name, and the inner bytestring is the rowkey. this is no longer obvious without reading the code where this is used, because you are no longer using the type that had that documentation.

Reply:
Yes, agreed. I think just giving them `type` aliases alone would help tremendously.